package.json

```diff
@@ -41,8 +41,8 @@
     "node": ">=18"
   },
   "dependencies": {
-    "color-convert": "^3.0.1",
-    "color-string": "^2.0.0"
```

Comment:
These should automatically get picked up with the ranges, no?

Reply:
Yes. The package manager will not follow the `package-lock.json` file, regardless of whether it's present or not.

Let me revert this change if you mind.

Reply:
Yep. I missing typed this. I cleared the field and forgot to add it back...